src/TNBase/Forms/FormMain.cs

```diff
@@ -261,7 +261,11 @@ private void BtnFinished_Click(object sender, EventArgs e)
                     warnings.Add("No recently added Listeners this week.");
                 }
 
-                if (serviceLayer.GetNextWeekBirthdays().Count > 0)
+                DateTime fromDate;
+                DateTime toDate;
+
+                serviceLayer.GetUpcomingBirthdayDates(out fromDate, out toDate);
```

Comment:
Just an FYI, you should be able to use an inline variable declaration like this `serviceLayer.GetUpcomingBirthdayDates(out DateTime fromDate, out DateTime toDate);` or simply with vars like this `serviceLayer.GetUpcomingBirthdayDates(out var fromDate, out var toDate);`.

However, as I've mentioned above, I'd prefer to have something like this:
```suggestion
                var dateRange = serviceLayer.GetUpcomingBirthdayDates();
                if (serviceLayer.GetUpcomingBirthdays(dateRange.From,  dateRange.To).Count > 0)
```